### ISSUE TITLE
Brokenspoke-analyzer error message

### ISSUE THREAD

**Issue by jaredcowing** [PeopleForBikes/brokenspoke-analyzer#279]
# Help request

After initial install of the brokenspoke-analyzer and running a test analysis, I get an error.

## Problem

After installing from GitHub directly and running a test command ```bna run massachusetts pittsfield``` I get this error:
```AttributeError: module 'brokenspoke_analyzer.core.analysis' has no attribute 'download_lodes_data'```
Originating from line 291 of cli.py.
Being new to this analyzer I am probably missing something basic, but thought I would ask to see if this has a simple solution.

I can alternatively try to do a Docker install, though I don't yet have experience with Docker (I'm not a software developer) so am curious if it should still be possible to run an analysis using the simpler install without this particular error.

Thanks for helping a newbie.

**Comment by jaredcowing** [comment]
I apologize, I think I just misread the first time around, am I correct in understanding that a Docker image is required to run the analysis?

**Comment by rgreinho** [comment]
Thanks a lot for reporting the problem. 

It looks like you were using our `main` branch, which we use for development. We try to keep it bug free, but there is no guarantee. However we do provide tagged versions that should not contain this bug. We recommend you use `v1.2.1`, or if you are not in a hurry, we plan on releasing `v1.3.0` in the coming weeks (maybe days).

You are absolutely correct, Docker will be required to complete the process. As well as osmium. See the [official documentation](https://peopleforbikes.github.io/brokenspoke-analyzer/#requirements) for reference.

Basically the process is composed of 2 steps:
- Step 1, which will collect all the data required to perform the analysis.
- Step 2 which will perform the analysis. This happens inside the Docker container and leverages PostgreSQL/PostGIS.

**Comment by rgreinho** [comment]
@jaredcowing We just released a new version: 1.3.0. We believe this version will solve the issue you encounter.

**Comment by jaredcowing** [comment]
@rgreinho  Thanks! I'll give it a try. Thank you also for the clarification in the documentation. I appreciate your help.
